go/cmd/dolt/commands/blame.go

```diff
@@ -143,6 +144,25 @@ func (cmd BlameCmd) Exec(ctx context.Context, commandStr string, args []string,
 	return 0
 }
 
+type RowMap struct {
+	Key   string
+	Value []interface{}
+}
+
+type RowMapList []RowMap
+
+func (p RowMapList) Len() int      { return len(p) }
+func (p RowMapList) Swap(i, j int) { p[i], p[j] = p[j], p[i] }
+func (p RowMapList) Less(i, j int) bool {
+	if len(p[i].Key) < len(p[j].Key) {
+		return true
+	} else if len(p[i].Key) == len(p[j].Key) {
```

Comment:
I'm not sure I understand, why are you comparing the length of the key? aaron comes before ale.